examples/release_highlights/plot_release_highlights_1_5_0.py

```diff
@@ -0,0 +1,182 @@
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+
+"""
+
+# %%
+# FixedThresholdClassifier: Setting the decision threshold of a binary classifier
+# -------------------------------------------------------------------------------
+# All binary classifiers of scikit-learn use a fixed decision threshold of 0.5 to
+# convert probability estimates (i.e. outputs of `predict_proba`) into class
+# predictions. However it is often not the desired threshold for a given problem.
+# :class:`~model_selection.FixedThresholdClassifier` allows to wrap any binary
+# classifier and set a custom decision threshold.
+from sklearn.datasets import make_classification
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+X, y = make_classification(n_samples=1_000, weights=[0.9, 0.1], random_state=0)
+classifier = LogisticRegression(random_state=0).fit(X, y)
+
+print("confusion matrix:\n", confusion_matrix(y, classifier.predict(X)))
+
+# %%
+# Moving the threshold towards the negative class, i.e. allowing more samples to be
+# classified as the positive class, increases the number of true positives at the
+# cost of more false positives.
+from sklearn.model_selection import FixedThresholdClassifier
+
+wrapped_classifier = FixedThresholdClassifier(classifier, threshold=0.1).fit(X, y)
+
+print("confusion matrix:\n", confusion_matrix(y, wrapped_classifier.predict(X)))
+
+# %%
+# TunedThresholdClassifierCV: Tuning the decision threshold of a binary classifier
+# --------------------------------------------------------------------------------
+# The decision threshold of a binary classifier can be tuned to optimize a given
+# metric, using :class:`~model_selection.TunedThresholdClassifierCV`.
+from sklearn.metrics import balanced_accuracy_score
+
+# Due to the class imbalance, the balanced accuracy is not optimal for the default
+# threshold. The classifier tends to over predict the majority class.
+print(f"balanced accuracy: {balanced_accuracy_score(y, classifier.predict(X)):.2f}")
+
+# %%
+# Tuning the threshold to optimize the balanced accuracy gives a smaller threshold
+# that allows more samples to be classified as the positive class.
+from sklearn.model_selection import TunedThresholdClassifierCV
+
+tuned_classifier = TunedThresholdClassifierCV(
+    classifier, cv=5, scoring="balanced_accuracy"
+).fit(X, y)
+
+print("new threshold: {tuned_classifier.best_threshold_:.4f}")
+print(
+    "balanced accuracy: {balanced_accuracy_score(y, tuned_classifier.predict(X)):.2f}"
+)
+
+# %%
+# :class:`~model_selection.TunedThresholdClassifierCV` also benefits from the
+# metadata routing support allowing to optimze complex business metrics, detailed
+# in :ref:`Post-tuning the decision threshold for cost-sensitive learning
+# <sphx_glr_auto_examples_model_selection_plot_cost_sensitive_learning.py>`.
+
+# %%
+# Performance improvements in PCA
```

Comment:
Not actionable: I know we are a bit thin with big new things and this is one of them. But we did not list the newton-cholesky solver in 1.2 as a highlight.

Reply:
I think we should have promoted the newton-cholesky in 1.2.

If I put on my "developer relationships" hat on, I would add a new entry for newton-cholesky solver in the 1.2 highlights and link to it in the 1.5 highlights.